<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoHideConfig</class>
 <widget class="QWidget" name="AutoHideConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>253</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Jobs &amp; Notifications</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoHide">
     <property name="text">
      <string>Automatically hide new notifications and jobs after a short delay</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>System Tray Icons</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="icons"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
